src/VirtualClient/VirtualClient.Actions/ApacheBench/ApacheBenchExecutor.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Actions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO.Abstractions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// Apache http server benchmarking workload executor
+    /// </summary>
+    public class ApacheBenchExecutor : VirtualClientComponent
```

Comment:
For workload like this, can we use server/client to test the network?

Reply:
To be merged as separate PR.